server/words.js

```diff
@@ -8,53 +8,57 @@ const easy = ['cat', 'dog', 'umbrella', 'mouse', 'apple', 'banana', 'pineapple']
 const medium = ['merry-go-round', 'polaroid', 'photo album']
 const hard = ['high school', 'carnival', 'castle']
 const impossible = ['mystery', 'orange', 'novel', 'regret', 'excited']
-const custom = {}
+// const custom = {}
 
-const words = { easy, medium, hard, impossible, custom }
+const DEFAULT_WORDS = { easy, medium, hard, impossible, custom: [] }
+
+const wordSets = {}
 
 function addCustomWord(roomCode, word) {
-  if (Object.prototype.hasOwnProperty.call(custom, roomCode)) {
-    if (!custom[roomCode].find((current) => current === word)) {
-      custom[roomCode] = [...custom[roomCode], word]
-    }
-    console.log('if room already exists', Object.values(custom))
-  } else {
-    custom[roomCode] = [word]
-    console.log('room doesnt exists', Object.values(custom))
+  if (!wordSets[roomCode].custom.includes(word)) {
+    wordSets[roomCode].custom = [...wordSets[roomCode].custom, word]
```

Comment:
If I'm understanding this correctly, we're trying to add a new word to the custom list within the owrdSets object right? Would we not be able to do something like wordSets[roomCode].custom.push(word) instead?